src/devices/Arduino/ArduinoBoard.cs

```diff
@@ -88,6 +88,17 @@ public ArduinoBoard(string portName, int baudRate)
         /// </summary>
         protected ILogger Logger => _logger;
 
+        /// <summary>
+        /// Set this to true if the underlying stream uses some kind of hardware or low-level flow control (RTS/CTS for
+        /// a serial port, or a TCP socket). Setting this to true may improve performance on bulk transfers (such as
+        /// large SPI blocks) but can result in buffer overflows if flow control is not working. Default: false
+        /// </summary>
+        public bool StreamUsesHardwareFlowControl
```

Comment:
Should this be readonly and set in the constructor though? Or do we want to allow it being mutable?

Reply:
Correct, this should be defined in the ctor. Changing it at runtime will likely cause problems.